packages/slice-machine/server/src/api/index.ts

```diff
@@ -174,13 +175,13 @@ router.get(
     >,
     res: express.Response
   ): Promise<Express.Response> {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
     const payload = await pushSlice(req.query);
```

Comment:
can this throw?

Reply:
There's a try catch inside the pushSlice (the one on this line is a default export) that is used at multiple places, it returns an "OnError".